.github/workflows/end-to-end-test-portals.yml

```diff
@@ -0,0 +1,86 @@
+name: End to end testing for Portals
```

Comment:
Since this workflow only applies to Portals, I kept e2e testing separate from the existing build-test.yml workflow. Happy to fold into that workflow, if that'd be preferred though!

Reply:
If it gets complex enough, refactoring to multiple yml files seem to make sense.